### PR TITLE
Add a tutorial for using the pre-trained models.

### PR DESCRIPTION
**Pull request by schmmd**

### CODE REVIEW

**Comment by schmmd** [comment]
The content isn't exactly earth shattering, but this directly addresses a use case that our tutorials don't quite cover.

File: tutorials/getting_started/using_pretrained_models.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+```bash
+python -m allennlp.run predict \
+    https://s3-us-west-2.amazonaws.com/allennlp/models/ner-model-2018.04.26.tar.gz \
+    ner-examples.jsonl \
+    --predictor sentence-tagger
+```
+
+## Making Predictions Programatically
+
+You can also make productions from python, using AllenNLP as a library.  The following code example
```
Comment (joelgrus) [inline tutorials/getting_started/using_pretrained_models.md]: typo: predictions

File: tutorials/getting_started/using_pretrained_models.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+uses the default predictor (`sentence-tagger`) for the NER model.
+
+```python
+from allennlp.service.predictors import Predictor
+predictor = Predictor.from_path("https://s3-us-west-2.amazonaws.com/allennlp/models/ner-model-2018.04.26.tar.gz")
+results = predictor.predict("Did Uriah honestly think he could beat The Legend of Zelda in under three hours?")
```
Comment (joelgrus) [inline tutorials/getting_started/using_pretrained_models.md]: I would use named parameters here `sentence="Did Uriah..."` and call out explicitly that the parameters to `predict` depend on which predictor you're using

File: tutorials/getting_started/using_pretrained_models.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+for word, tag in list(zip(results["words"], results["tags"])):
```
Comment (joelgrus) [inline tutorials/getting_started/using_pretrained_models.md]: you don't need a `list` here, you can iterate over `zip` without it

File: tutorials/getting_started/using_pretrained_models.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    print(f"{word}\t{tag}"
+```
+
+And the sample output:
+
+```
+['O', 'U-PER', 'O', 'O', 'O', 'O', 'O', 'B-MISC', 'I-MISC', 'I-MISC', 'L-MISC', 'O', 'O', 'O', 'O', 'O']
```
Comment (joelgrus) [inline tutorials/getting_started/using_pretrained_models.md]: how does this get output? I don't see the print for it

File: tutorials/getting_started/using_pretrained_models.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
+# Using Pretrained AllenNLP Models
+
+In this tutorial, we show how to use run the pretrained models in AllenNLP to make predictions.
+While this tutorial will specifically use the Named Entity Recognizer, but will apply to any of
+the models [available on our website](http://allennlp.org/models).
```
Comment (joelgrus) [inline tutorials/getting_started/using_pretrained_models.md]: this sentence is awkward

"This tutorial uses the Named Entity Recognition model, but the same procedure applies to any of the model available on our website"

File: tutorials/getting_started/using_pretrained_models.md (hunk 4 above)
Comment (schmmd) [reply tutorials/getting_started/using_pretrained_models.md]: Accident--that was the old output.

**Comment by schmmd** [comment]
@joelgrus I responded to your feedback.

**Comment by matt-gardner** [comment]
Nit: this seems more like a "how-to" than a "tutorial" to me - does this fit better in that section?

**Comment by schmmd** [comment]
It's very much a `how_to` but it also falls squarely under `getting_started`.  I think a number of users want to get started using AllenNLP this way.  Perhaps we should re-work our categories...

**Pull request merged**